MapPointMapper/Parser/Parser.swift

```diff
@@ -28,6 +32,16 @@ extension String {
 
 class Parser {
     // MARK: - Public
+    /**
+    Parse a given string of Lat/Lng values to return a collection of `CLLocationCoordinate2D` arrays.
+
+    :note: The preferred way/format of the input string is `Well-Known Text` as the parser supports that for multipolygons and such
+
+    :param: input          String to parse
+    :param: longitudeFirst Only used if it is determined to not be `Well-Known Text` format.
```

Comment:
Well, it can still be input without being formatted as such. Like stripping off `LINE` and just passing in a collection of points to the text box will still work correctly.